src/Microsoft.CodeAnalysis.Analyzers/UnitTests/MetaAnalyzers/AddLanguageSupportToAnalyzerRuleTests.cs

```diff
@@ -55,15 +55,15 @@ Imports Microsoft.CodeAnalysis.Diagnostics
 
 <DiagnosticAnalyzer(LanguageNames.VisualBasic, ""MyLanguage"")>
 Class MyAnalyzer
-	Inherits DiagnosticAnalyzer
-	Public Overrides ReadOnly Property SupportedDiagnostics() As ImmutableArray(Of DiagnosticDescriptor)
-		Get
-			Throw New NotImplementedException()
-		End Get
-	End Property
-
-	Public Overrides Sub Initialize(context As AnalysisContext)
-	End Sub
+    Inherits DiagnosticAnalyzer
+    Public Overrides ReadOnly Property SupportedDiagnostics() As ImmutableArray(Of DiagnosticDescriptor)
+        Get
+            Throw New NotImplementedException()
+        End Get
+    End Property
+
+    Public Overrides Sub Initialize(context As AnalysisContext)
+    End Sub
```

Comment:
⚠️ All of this indentation should not have changed. The indentation is located within string literals. Applies to multiple files.